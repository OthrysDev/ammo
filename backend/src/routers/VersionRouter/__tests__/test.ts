import app from 'index';
import request from 'supertest';

afterAll(() => {
    app.close();
});

describe('VersionRouter', () => {
    describe('get("/")', () => {
        test('Get version - should return version', async () => {
            await request(app)
                .get('/version')
                .expect(200)
                .expect((r) => {
                    expect(r.body).toMatchSnapshot();
                });
        });
    });
});
