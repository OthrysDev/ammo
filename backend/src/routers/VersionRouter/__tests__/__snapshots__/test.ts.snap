// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VersionRouter get("/") Get version - should return version 1`] = `
Object {
  "version": "0.0.0",
}
`;
